adoc/admin-troubleshooting.adoc

```diff
@@ -39,15 +43,49 @@ cd /var/log/nts*
 sudo cat kubernetes.txt crio.txt
 ----
 
-The result is a TAR archive of files. Each of the tar files should be given a name that can be used to identify which
+The result is a `TAR` archive of files. Each of the `*.txz` files should be given a name that can be used to identify which
 cluster node it was created on.
 
 After opening a Service Request (SR), you can upload the TAR archives to Global Technical Support.
 
-The data will help to locate the issue you reported and assist you in solving the problem.
+The data will help to debug the issue you reported and assist you in solving the problem.
 For details, see https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_adm_support.html.
 
-== Debugging Cluster Deployment Failed
+== Cluster definition directory
+
+Apart from the logs provided by running the `supportconfig` tool, an additional set of data might be required for
+debugging purposes. This information is located at the Management node, under your cluster definition directory.
+This folder contains important and sensitive information about your CaaSP cluster and it's the one
+from where you issue `skuba` commands.
+
+[TIP]
+====
+If the problem you are facing is related to your production environment, do **not** upload the `admin.conf` as this would
```

Comment:
We should also advise to not upload the `pki` directory, since this also contains sensitive information (ca cert and key).

Reply:
I forgot about it. Thanks for the sharp eye ;)